server/src/main/java/org/opensearch/index/remote/RemoteSegmentStats.java

```diff
@@ -91,6 +99,8 @@ public RemoteSegmentStats(StreamInput in) throws IOException {
          */
         if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
             totalRefreshBytesLag = in.readLong();
+            totalUploadTime = in.readLong();
+            totalDownloadTime = in.readLong();
```

Comment:
Dont we need to first have the comparison on current version, backport and then do the change of exact version & backport?

Reply:
Marking the version check with `Version.CURRENT` to ensure BwC. Will remove this compatibility check once the code is backported to 2.x